### DISCUSSION TITLE
Is there a way to cover swift files?

### DISCUSSION THREAD

**Discussion by faisalmohd83** [insideapp-oss/sonar-flutter#145] [Q&A]
Hi, 

We got swift code in our multi-module project and could not scan those files. Is there a way to cover it? 
Our project folder has `packages/` with multiple sub-modules with native code. Dart/kotlin files are scanned but not the swift ones.

sonar-project.properties is as below:

```
sonar.sources=android/app/src, ios, lib, packages
sonar.exclusions=**/*.pb.swift, android/app/src/**/res/**
sonar.language=dart, swift, kt
```

**Comment by zippy1978** [comment]
Hi @faisalmohd83,

To cover Swift file, you can use this SonarQube plugin: https://github.com/insideapp-oss/sonar-apple
